.github/workflows/run-legoschema.yml

```diff
@@ -0,0 +1,102 @@
+name: Run Legoschema
+on:
+  pull_request:
+    types: [opened, reopened, edited, ready_for_review]
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  USERNAME: "jayasimha-raghavan-unskript"
```

Comment:
Can we not a user and instead use some bot user?

Reply:
Done. Using Secrets->Build_user (a bot user)